packages/@react-native-windows/cli/src/runWindows/utils/winappdeploytool.ts

```diff
@@ -19,8 +19,8 @@ class DeviceInfo {
     public readonly guid: string,
     public readonly ip: string,
 
-    private index: number,
-    private type: string,
+    private readonly index: number,
+    private readonly type: string,
```

Comment:
Thanks love this change too!

Reply:
I was actually one the fence about removing this one. Want to play around a bit, but this one is auto-fixed on save, which means readonly will get automatically applied pretty much everywhere, even if it will eventually not be readonly after typing a few more lines.

Could see us turning it off if it gets too annoying though.

Reply:
Hmmmm, I think I remember having an opinion about auto-format on save 😏 😃